Sources/AppStoreConnectCLI/Commands/Capability/DisableBundleIdCapabilityCommand.swift

```diff
@@ -0,0 +1,27 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import AppStoreConnect_Swift_SDK
+import ArgumentParser
+
+struct DisableBundleIdCapabilityCommand: CommonParsableCommand {
+
+    public static var configuration = CommandConfiguration(
+        commandName: "disable",
+        abstract: "Disable a capability for a bundle ID."
+    )
+
+    @OptionGroup()
+    var common: CommonOptions
+
+    @Argument(help: "The reverse-DNS bundle ID identifier to delete. Must be unique. (eg. com.example.app)")
+    var bundleId: String
+
+    @Argument(help: "Bundle Id capability type. \n \(CapabilityType.allCases)")
```

Comment:
Use the extended description (or `discussion`) if you need to have this go over multiple lines. I would not recommend adding in a `\n`.